src/main/java/io/quarkus/search/app/indexing/FailureCollector.java

```diff
@@ -65,19 +66,21 @@ public record Failure(Level level, Stage stage, String details, Exception except
                 .thenComparing(f -> System.identityHashCode(f.exception()));
     }
 
+    protected final Clock clock;
```

Comment:
seems like this clock is never used, since we only pass it to the GH reporter in the constructor

Reply:
Fixed, thanks